walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -127,7 +99,9 @@
       }
       startOfPossibleTimes = timerange.end();
     }
-
+   
+    int endOfPossibleTimes = TimeRange.END_OF_DAY;
```

Comment:
This would probably make more sense to put on line 89 (just before the loop), just so you don't have to keep re-initializing the same variable over and over.